### PR TITLE
object deletion

### PR DESCRIPTION
**Pull request by aprasolova**

### CODE REVIEW

**Comment by realloc** [comment]
Previously we agreed to use line-per-paragraph format. Please change it.

File: 03-storage/09-object-deletion.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+### Object Deletion
+
+We cannot guarantee permanent object removal in distributed system.
```
Comment (EliChin) [inline 03-storage/09-object-deletion.md:3]: It's hard to guarantee complete object removal in a distributed system.

File: 03-storage/09-object-deletion.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Instead, Storage Node designed to be self-cleaning. Object Garbage Collector component is responsible for objects deletion physically from Blobstors and Metabase. Object Garbage Collector also serves the Control API endpoint for external data controllers and allows them make decisions on object deletion basing on their specific criterias. It runs periodically, classifies objects and selects them for removal.
```
Comment (EliChin) [inline 03-storage/09-object-deletion.md:5]: That is why our Storage Node is designed to be self-cleaning.

Comment (EliChin) [inline 03-storage/09-object-deletion.md:5]: Object Garbage Collector component deletes objects physically from Blobstors and Metabase. Plus, it exposes the Control API endpoint to external data controllers and allows them to make decisions on object deletion regarding applicable specific criteria.

**Review by realloc — APPROVED** [review]

**Review by realloc — APPROVED** [review]

**Pull request merged**